tests/seeyou/test_reader.py

```diff
@@ -10,12 +10,18 @@
 
 FOLDER = path.dirname(path.realpath(__file__))
 DATA_PATH = path.join(FOLDER, 'data', 'SEEYOU.CUP')
+CUPFILE_WAYPOINTS = path.join(FOLDER, 'data', 'WAYPOINTS.CUP')
 
 if_data_available = pytest.mark.skipif(
     not path.exists(DATA_PATH),
     reason="requires SEEYOU.CUP"
 )
 
+if_waypoints_cup_available = pytest.mark.skipif(
+    not path.exists(CUPFILE_WAYPOINTS),
+    reason="requires WAYPOINTS.CUP"
+)
+
```

Comment:
actually this shouldn't be needed. `@if_data_available` existed because at somepoint `SEEYOU.CUP` was not part of the repository and was downloaded by the CI server on demand. the new test based on `WAYPOINTS.CUP` (please rename to `waypoints.cup`, thanks) however should *always* run and there shouldn't be any way for the file to be missing.